lib/krane/kubernetes_resource.rb

```diff
@@ -414,9 +455,12 @@ def self.go_template_for(kind, name)
           '(ne .reason "Pulling")',
           '(ne .reason "Pulled")',
         ]
-        condition_start = "{{if and #{and_conditions.join(' ')}}}"
+
+        and_conditions_string = and_conditions.map { |c| "{{if #{c}}}" }.join("")
+        ends = "{{end}}" * and_conditions.length
+
         field_part = FIELDS.map { |f| "{{#{f}}}" }.join(%({{print "#{FIELD_SEPARATOR}"}}))
-        %({{range .items}}#{condition_start}#{field_part}{{print "#{EVENT_SEPARATOR}"}}{{end}}{{end}})
+        %({{range .items}}#{and_conditions_string}#{field_part}{{print "#{EVENT_SEPARATOR}"}}#{ends}{{end}})
```

Comment:
Yes, the older version of Kubectl (which we still support) are compiled with Go 1.17. Only since 1.18 it evaluates the statements correctly. 

https://tip.golang.org/doc/go1.18#:~:text=test.fuzzminimizetime.-,text/template,-Within%20a%20range